recipes/sktime/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "sktime" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/alan-turing-institute/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ac581f3cd3d0c17e81ac83c827ca1db7b2938d6d60deb13835c95dfd0913114a
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  run_exports:
+    - matplotlib >=3.3.2
+    # - pmdarima >=1.8.0
+    - scikit-posthocs >= 0.6.5
+    - seaborn >=0.11.0
+    - tsfresh >=0.17.0
+    # - catch22 >=0.1.0
+    - hcrystalball >=0.1.9
+    - stumpy >=1.5.1
+    # - tbats >=1.1.0
+    - fbprophet >=0.7.1
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+  host:
+    - python
+    - cython >=0.29
+    - numba >=0.50,<0.51
+    - numpy >=1.19
```

This will make it only possible to use the latest numpy version, using the global pin (1.16):
```suggestion
    - numpy
```
or
```suggestion
    - numpy =1.19
```
would be better

---

That's the version pinning from upstream. Maybe they use some 1.19 specific features - I'm not sure. If they don't, they certainly shouldn't be requiring the use of the latest `numpy` but I don't know enough about the project to be confident changing their claimed minimum versions.

---

https://github.com/alan-turing-institute/sktime/blob/c57d8829005c4bcd28a2c2bd1816e5ceb6734617/setup.py#L23-L29

---

I figured it's easy enough to relax that later (if possible) and publish a new build number.

---

We're happy to relax requirements where possible

---

> We're happy to relax requirements where possible

Since we're not running tests here I think that's probably best done as a PR to `sktime` (as with my other one) so that it gets tested first. If it passes CI then it's easy to relax the pin and publish a new build.

---

you still should have a lower bound like `numpy =1.19` instead of `numpy >=1.19`